demo/loaderUtilities.js

```diff
@@ -34,6 +34,17 @@ export const getFromRestApi = async (url) => {
   return await response.json();
 }
 
+export const setRestApi = async (url) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  });
+
+  return await response.json();
```

Comment:
Generally, `return await ...` is redundant, and equivalent to just `return ...`. I Don't mind if you feel it's clearer this way.